app/assets/stylesheets/ama_layout/foundation_and_overrides_battery.scss

```diff
@@ -0,0 +1,7 @@
+@charset 'utf-8';
```

Comment:
It's referenced here: https://github.com/amaabca/javascript_widgets/blob/reduced-css/app/assets/stylesheets/battery_quote/style.scss.erb